docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md

```diff
@@ -0,0 +1,119 @@
+
+## Predicted Entities
+
+`true`, `false`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/rct_binary_classifier_biobert_en_3.4.2_3.0_1653668780966.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler()\
+        .setInputCol("text")\
+        .setOutputCol("document")
+
+bert_sent = BertSentenceEmbeddings.pretrained("sent_biobert_pubmed_base_cased", "en") \
+        .setInputCols("document") \
+        .setOutputCol("sentence_embeddings")
+
+classifier_dl = ClassifierDLModel.pretrained("rct_binary_classifier_biobert", "en", "clinical/models")\
+        .setInputCols(["sentence_embeddings"])\
+        .setOutputCol("class")
+        
+biobert_clf_pipeline = Pipeline(
+    stages = [
+        document_assembler,
+        bert_sent, 
+        classifier_dl
+    ])
+
+data = spark.createDataFrame([["""Abstract:Based on the American Society of Anesthesiologists' Practice Guidelines for Sedation and Analgesia by Non-Anesthesiologists (ASA-SED), a sedation training course aimed at improving medical safety was developed by the Japanese Association for Medical Simulation in 2011. This study evaluated the effect of debriefing on participants' perceptions of the essential points of the ASA-SED. A total of 38 novice doctors participated in the sedation training course during the research period. Of these doctors, 18 participated in the debriefing group, and 20 participated in non-debriefing group. Scoring of participants' guideline perceptions was conducted using an evaluation sheet (nine items, 16 points) created based on the ASA-SED. The debriefing group showed a greater perception of the ASA-SED, as reflected in the significantly higher scores on the evaluation sheet (median, 16 points) than the control group (median, 13 points; p < 0.05). No significant differences were identified before or during sedation, but the difference after sedation was significant (p < 0.05). Debriefing after sedation training courses may contribute to better perception of the ASA-SED, and may lead to enhanced attitudes toward medical safety during sedation and analgesia. """]]).toDF("text")
+
+result = biobert_clf_pipeline.fit(data).transform(data)
+```
+```scala
+val documenter = new DocumentAssembler() 
+    .setInputCol("text") 
+    .setOutputCol("document")
+
+val bert_sent = new BertSentenceEmbeddings.pretrained("sent_biobert_pubmed_base_cased", "en")
+    .setInputCols("document")
+    .setOutputCol("sentence_embeddings")
+
+val classifier_dl = new ClassifierDLModel.pretrained("rct_binary_classifier_biobert", "en", "clinical/models")
```

Comment:
also this one

Reply:
I've just fixed it, thanks for the feedback!